src/Authentication/Authentication/custom/Find-MgGraphPermission.ps1

```diff
@@ -0,0 +1,226 @@
+6931bccd-447a-43d1-b442-00a195474933 Admin   MailboxSettings.ReadWrite Allows the app to create, read, update, and del…
+
+PS> Connect-MgGraph -Scopes MailboxSettings.Read
+
+In this example the value 'mailbox' was specified for the SearchString parameter. The resulting output showed the list
+of permissions grouped by Delegated or Application permission type. By viewing the output, the user is able to see that the
+desired delegated permission is 'MailboxSettings.Read'. The user then invokes the Connect-MgGraph command specifying that
+permission name for the value of the Scopes parameter in order to request the permission as part of a new sign-in.
+Subsequent PowerShell commands issued in the session that access Microsoft Graph will then be authorized with the permission
+if the sign-in was successful.
+
+.EXAMPLE
+PS> Find-MgGraphPermission User.Read -ExactMatch -PermissionType Delegated
+
+   PermissionType: Delegated
+
+Id                                   Consent Name      Description
+--                                   ------- ----      -----------
+e1fe6dd8-ba31-4d61-89e7-88639da4683d User    User.Read Allows you to sign in to the app with your organizational accou…
+
+In this example the PermissionType parameter restricts the output to only delegated permissions, and by specifying the ExactMatch
+parameter only permissions that exactly match the specified name are emitted.
+
+.EXAMPLE
+PS> 'User.Read.All', 'Group.Read.All' | Find-MgGraphPermission -ExactMatch -PermissionType Application | Select-Object Id
+
+Id
+--
+df021288-bdef-4463-88db-98f22de89214
+5b567255-7703-4780-807c-7be8301ae99b
+
+This example demonstrates that ability to pass the SearchString parameter as pipeline input. In this case, the issued command
+returns the unique identifiers of the two permissions listed in the pipeline, 'User.Read.All' and 'Group.Read.All'. The
+PermissionType parameter was included with a value of Application to ensure that only application permissions were emitted,
+and ExactMatch ensures that the intent of emitting output just for these specific permissions and not some that might match
+their names partially is honored.
+
+.EXAMPLE
+Find-MgGraphPermission mailbox | Where-Object PermissionType -eq Delegated | Format-List Name, Description
+
+Name        : MailboxSettings.Read
+Description : Allows the app to read your mailbox settings.
+
+Name        : MailboxSettings.ReadWrite
+Description : Allows the app to read, update, create, and delete your mailbox settings
+
+This example outputs the set of delegated permissions that match the 'mailbox' SearchString parameter. The Where-Object
+command is used to filter the permissions to only the delegated permissions through the PermissionType property. The filtered
+result is piped to the Format-List command so that the output of the Description field is not truncated as it would be
+in the default table view.
+
+.LINK
+https://docs.microsoft.com/en-us/graph/permissions-reference
```

Comment:
This allows `Get-Help -Online Find-MgGraphPermission` to actually point at the permissions reference rather than help for `Find-MgGraphPermission`. If that's not ok, we can change this.